### PR TITLE
Rnr 68 test setup

### PR DESCRIPTION
**Pull request by jack829**
## JIRA Link
https://scrychat.atlassian.net/browse/RNR-68

## Changes
* Add poc unit tests on actions
* Add travis-ci hookup

## Other Dependencies
none

## ScreenShots
none

### CODE REVIEW

**Comment by dblanchard13** [comment]
screen shot 2018-12-04 at 3 10 44 pm
Can see the build deets ^^

File: babel.config.js [hunk 1]
```diff
@@ -4,8 +4,11 @@ module.exports = function(api) {
...
     env: {
       development: {
-        plugins: ["@babel/plugin-proposal-class-properties", "@babel/transform-react-jsx-source"]
-      }
-    }
-  };
-};
+        plugins: [
+          "@babel/plugin-proposal-class-properties",
+          "@babel/transform-react-jsx-source"
```
Comment (dblanchard13) [inline babel.config.js]: NIT: dangler 🙃

Comment (jack829) [reply babel.config.js]: have we been dangling arrays?

**Comment by dblanchard13** [comment]
screen shot 2018-12-04 at 3 12 34 pm

**Pull request merged**